components/util/socialIcons.tsx

```diff
@@ -34,56 +34,56 @@ const socialStyles = new Map<
     SocialTypes.phone,
     {
       icon: FaPhone,
-      bgClassName: "bg-phone",
+      bgClassName: "bg-socialPhone",
     },
   ],
   [
     SocialTypes.youtube,
     {
       icon: FaYoutube,
-      bgClassName: "bg-youtube",
+      bgClassName: "bg-socialYoutube",
     },
   ],
   [
     SocialTypes.linkedin,
     {
       icon: FaLinkedinIn,
-      bgClassName: "bg-linkedin",
+      bgClassName: "bg-socialLinkedin",
     },
   ],
   [
     SocialTypes.facebook,
     {
       icon: FaFacebookF,
-      bgClassName: "bg-facebook",
+      bgClassName: "bg-socialFacebook",
     },
   ],
   [
     SocialTypes.instagram,
     {
       icon: FaInstagram,
-      bgClassName: "bg-instagram",
+      bgClassName: "bg-gradient-to-tr from-[#f09433] to-[#bc1888]",

```

Comment:
would be good if this referenced a tailwind class(es) like the others

try using https://github.com/benface/tailwindcss-gradients to replicate the gradient